packages/web-components/fast-foundation/README.md

```diff
@@ -0,0 +1,2 @@
+# FAST Foundation
+A suite of templates, classes, and utilities for composing web components.
```

Comment:
Can we make a note that this should be fleshed out like our other README files to show installation and basic use?

Reply:
Yeah I wanted to get feedback on what all we want to outline in here.